web/src/elements/sidebar/SidebarItem.ts

```diff
@@ -144,47 +144,84 @@ export class SidebarItem extends AKElement {
         return this.renderInner();
     }
 
-    renderInner(): TemplateResult {
-        if (this.childItems.length > 0) {
-            return html`<li
-                class="pf-c-nav__item ${this.expanded ? "pf-m-expandable pf-m-expanded" : ""}"
+    renderWithChildren() {
```

Comment:
The four objects here represent the four main items drawn by the sidebar.  We don't yet have the `withPathAndChildren()` yet; that's for when we get to the subtypes.